.travis.yml

```diff
@@ -12,14 +12,18 @@ before_deploy: |
   if [[ $TRAVIS_HASKELL_VERSION == "8.6.5" ]]; then
     cabal install -f 'static'
     tar -C dist/build/dotenv -cvf dotenv-linux-x86_64-static.tar.gz dotenv
+    curl https://github.com/lindell/github-release-cli/releases/download/1.3.0/github-releaser-travis -L --output github-releaser && chmod +x github-releaser
```

Comment:
@CristhianMotoche I'm not against using a third-party tool here, but I think `curl` could get the job done. My concern about bringing up another tool is that we have to take care of keeping it up to date and install all required dependencies, compared to `curl` which is a system dependency that could be easily updated.

Reply:
Make sense @sestrella Let me take a look at the GitHub API for that.